Readme.md

```diff
@@ -1,25 +1,32 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128564818/21.2.5%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E3050)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
-* [Default.aspx](./CS/Default.aspx)
-* [Default.aspx.cs](./CS/Default.aspx.cs) 
-<!-- default file list end -->
-# How to organize Wizard interface within ASPxPageControl
+# Page Control for ASP.NET Web Forms - How to create a wizard interface
 <!-- run online -->
 **[[Run Online]](https://codecentral.devexpress.com/e3050/)**
 <!-- run online end -->
 
+This example demonstrates how to use the [Page Control](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPageControl) to create a wizard interface.
+
+![Create a Wizard](create-a-wizard.png)
+
+In the example, a step is a [tab page](https://docs.devexpress.com/AspNet/DevExpress.Web.TabPage) that displays several fields. Mandatory fields are marked with an asterisk. Fill in the fields and click the **Next** button at the bottom of the current page to proceed to the next step. Click the **Personal**, **Date**, or **Contact** tab to navigate to the corresponding step.
```

Comment:
> In the example, **a step** is a [tab page](https://docs.devexpress.com/AspNet/DevExpress.Web.TabPage) that displays **several** fields.

**First**. I understand that you intended to say that "any step is a tab page", which is correct. However, I feel a lack of clarity in this wording. What is important? The important thing that we speak of a wizard step. I believe we should explicitly mention this. The second important thing that every wizard step is a tab page in this example.
**Second**. Several. Several is a colloquialism, and we do not use colloquialisms in documentation. And no, "several" is not fewer than "multiple". The both words mean "two or more" in this context. Therefore, we use "multiple" everywhere we would have placed "several". No exceptions.
**Third**. To satisfy my curiosity: I should fill in all the fields on the current tab page to proceed further. Yet at the same time you say that I can click any of the named tabs to navigate to the corresponding step directly. Do I have to fill in all fields and pass validation before clicking a tab? Will a click on a different tab have any effect if I omitted a mandatory field on the current tab? The last sentence raises more questions than answers. There is a potential logical issue or lack of important information on how this example functions.

For now, I suggest the following edit, however, I cannot rectify the logical issue for you because I do not understand how this example works exactly.

```suggestion
In the example, every wizard step is a [tab page](https://docs.devexpress.com/AspNet/DevExpress.Web.TabPage) that displays multiple fields. Mandatory fields are marked with an asterisk. Fill in the fields and click the **Next** button at the bottom of the current page to proceed to the next step. Click the **Personal**, **Date**, or **Contact** tab to navigate to the corresponding step.
```

Please clarify.

Reply:
To move further, you must fill in all mandatory fields. But you can click the tab to go back, and in this case the validation is not performed, and the required fields may be empty.

Reply:
OK. Now I understand the way it works. Then we should explain exactly that and be done with it, don't you think?

Reply:
Can we say the following?

```suggestion
In the example, every wizard step is a [tab page](https://docs.devexpress.com/AspNet/DevExpress.Web.TabPage) that displays multiple fields. Mandatory fields are marked with an asterisk. You need to fill in the mandatory fields to move forward, but you can keep them empty to move backward. Click the **Next** button at the bottom of the current page to proceed to the next step, or click the **Personal**, **Date**, or **Contact** tab to navigate to the corresponding step.
```

Reply:
Yes. This makes more sense indeed.

"Backward" is a nominative form of a direction. When it comes to action, one moves backwards rather than backward.

```suggestion
In the example, every wizard step is a [tab page](https://docs.devexpress.com/AspNet/DevExpress.Web.TabPage) that displays multiple fields. Mandatory fields are marked with an asterisk. You need to fill in the mandatory fields to move forward but you can keep them empty to move backwards. Click the **Next** button at the bottom of the current page to proceed to the next step, or click the **Personal**, **Date**, or **Contact** tab to navigate to the corresponding step.```